src/content/docs/get-started/release.mdx

```diff
@@ -62,4 +62,4 @@ Slots are built-in extension points in the drop-in components to add your own UI
 
 #### Drop-in design tokens
 
-Drop-in [design tokens](/dropins/all/branding/#design-tokens) are defined in the drop-in SDK and are used by the drop-in components to style the storefront application.
\ No newline at end of file
+Drop-in [design tokens](/dropins/all/branding/#design-tokens) are defined in the drop-in SDK and are used by the drop-in components to style the storefront application.
```

Comment:
I didn't change anything about this line, not sure why it's showing up in the diff